### PR TITLE
doc / updated campaign changes for this week \

### PR DESCRIPTION
**Pull request by rapcmia**
Updated campaign changes for this week dec14-18

> Added  changes AVAX 
     - Added changes on weekly rewards
     - Added AVAX TRY

> Added changes on NEM
     - Added changes on weekly rewards
     - Added XEM/USDT

### CODE REVIEW

**Comment by rapcmia** [comment]
BXeXg4bkZB

File: content/liquidity-mining/current-rewards&terms.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ We will notify participants of changes, if any, on our [Discord](https://discord
 
 ## Current Reward Period
 
-**December 8, 2020 12:00am UTC - December 14, 2020 11:59pm UTC**
+**December 14, 2020 12:00am UTC - December 17, 2020 11:59pm UTC**
```
Comment (dennisocana) [inline content/liquidity-mining/current-rewards&terms.mdx:12]: December 15 to December 21

**Review by dennisocana — CHANGES_REQUESTED** [review]
See comment.

**Comment by rapcmia** [comment]
Request changes has been updated, thanks

**Pull request merged**